src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -190,267 +201,536 @@ class GitOpsConfiguration extends Component<GitOpsProps, GitOpsState> {
             },
             isError: {
                 ...this.state.isError,
-                [key]: event.target.value.length === 0 ? "This is a required field" : "",
+                [key]: event.target.value.length === 0 ? 'This is a required field' : '',
             },
             isFormEdited: false,
+            //After entering any text,if GitOpsFieldKeyType is of type host then the url validation error must dissapear
+            isUrlValidationError: key === 'host' ? false : this.state.isUrlValidationError,
         })
     }
 
     getFormErrors(isFormEdited, form: GitOpsConfig): any {
```

Comment:
this is convoluted logic, could have been done in simpler fashio

Reply:
done